source/_sass/infeedl.scss

```diff
@@ -27,7 +27,7 @@ h1, h2, h3, h4, h5, h6 {
 
 .navigation-wrapper {
   font-family: "gotham_proregular", Helvetica, Arial, sans-serif;
-  background: #6fc2d5;
+  background: #212121;
```

Comment:
Color literals like `#212121` should only be used in variable declarations; they should be referred to via variable everywhere else.